### ISSUE TITLE
Upgrade CLI to use v3.0

### ISSUE THREAD

**Issue by misshula** [Shopify/shopify-marketplaces-admin-app#17]
Our current docs use CLI v3.0 and this repo is built with v2.0

cc: @jeanrick

**Comment by cma93** [comment]
yeah probably that would resolve this issue too. https://github.com/Shopify/shopify-marketplaces-admin-app/issues/28

**Comment by cma93** [comment]
Would this include upgrading the shopify-api too?

**Comment by tomadelaney** [comment]
Apologies if my #37 is redundant.
Does Shopify have any plans to update the repo to CLI v3.0?

**Comment by MoSattler** [comment]
any news on this?

**Comment by agusqu** [comment]
Does anybody know if this repo is still being maintained by anyone? Has anyone been successful in fully migrating to CLI v3.0?

**Comment by tomadelaney** [comment]
Shopify support indicated in February 2023 that there were no immediate plans to update this repo to CLI v3.0.
